index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Hello Studio Website.">
+    <title>Hello Studio</title>
+  </head>
+  <body>
+    <header>
+      <h1>
+        <span>the time is</span>
+        <span>now</span>
+      </h1>
+      <nav>
+        <ul>
+          <li><a href="/">intro</a></li>
```

Comment:
These links correspond to the sections of the website, we can use anchor tags to make it interactive. So if the section for intro link had, say, id `#intro` we would have an `<a href="#intro">intro</a>` tag.

Reply:
👍